framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -3055,12 +3055,7 @@ private AnnotationMirror getDeclAnnotation(
 
             // Retrieving annotations from stub files.
             Set<AnnotationMirror> stubAnnos = stubTypes.getDeclAnnotation(elt);
-            if (stubAnnos != null) {
-                results.addAll(stubAnnos);
-            } else {
-                stubAnnos = stubTypes.getDeclAnnotation(elt);
```

Comment:
@smillst Should this maybe have been a call to something else?

Reply:
I think it was just a mistake.  I don't see anything else that this should be doing.